src/reducers/sortBy.js

```diff
@@ -0,0 +1,19 @@
+const sortBy = (state, action) => {
+  const sort = action.sortBy;
+  switch (sort) {
+    case 'releaseDay':
```

Comment:
sortBy reducer has to react on `SORT_BY` action and put new sort type to the state